src/web/releases.rs

```diff
@@ -1213,4 +1213,57 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn check_releases_page_content() {
```

Comment:
Isn't there already a test for this?

Reply:
I thought to but couldn't find it.

Reply:
Here it is: https://github.com/rust-lang/docs.rs/blob/d688dc78f5d6e63f07b7c40b275e77817ef466c5/src/web/releases.rs#L1024
So I think I would prefer to remove this duplicate test.

Reply:
It doesn't test the same things. My test added checks specifically to ensure the links were not removed by mistake (which I did while changing the templates when working on this PR).